File: pulumi/examples/python/digitalocean/digitalocean_spaces_example.py
"""
Pulumi DigitalOcean Spaces Bucket Example (Python)

Setup:
  pip install pulumi pulumi_digitalocean
Usage:
  pulumi up
"""
import pulumi
import pulumi_digitalocean as do

bucket = do.SpacesBucket('example-space',
    region='nyc3'
)
pulumi.export('bucket_name', bucket.name)

File: pulumi/examples/typescript/digitalocean_spaces_example.ts

# Pulumi DigitalOcean Spaces Example (Python)
import pulumi
import pulumi_digitalocean as digitalocean

example_space = digitalocean.SpacesBucket(
    "example-space",
    region="nyc3"
)

pulumi.export("bucket_name", example_space.name)
